src/ui_basic/box.cc

```diff
@@ -33,7 +33,8 @@ Box::Box(Panel* const parent,
          uint32_t const orientation,
          int32_t const max_x,
          int32_t const max_y,
-         uint32_t const inner_spacing)
+         uint32_t const inner_spacing,
+         bool const ignore_invisible_items)
```

Comment:
Is there a situation where we would not want to ignore them?

Reply:
No, I in general there should be no such situations. However, there are > 100 calls to Panel::setVisible() and for sure I didn't catch all of these during my limited testing. I found it safer to keep the Box' behaviour as before. But maybe it is more straight forward to always ignore invisible items and if some layouting bugs occur because of that, we will fix them...? 
(I'd prefer not to have the boolean flag)

Reply:
After some more testing, I removed the boolean flag. I did not encounter any problems so far :)